app_dart/lib/src/model/firestore/task.dart

```diff
@@ -377,3 +382,21 @@ Task taskToDocument(datastore.Task task) {
     ),
   );
 }
+
+/// A [Task], paired with its associated parent [Commit].
+///
+/// The [Task] model object references its parent [Commit] through the
+/// [Task.commitKey] field, but it does not hold a reference to the associated
+/// [Commit] object (just the relational mapping). This class exists for those
+/// times when the caller has loaded the associated commit from the datastore
+/// and would like to pass both the task its commit around.
```

Comment:
This needs an update. We have migrated to firestore, and there is no `Task.commitKey` connection any more.